artemis-chart/values.yaml

```diff
@@ -45,6 +45,7 @@ limitRequests: 20
 captchaWindow: 900000
 captchaTries: 4
 artemisWebBaseDir: ""
+serviceAccountPath: ""
```

Comment:
please also add to `.env` with proper name and also map in the docker-compose files so that the respective container(s) can access it. Same for k8s, the  containers that need it need to be able to access it via env mapping.

Reply:
Sorry, which .env file do you mean?

Cause I included the var as SERVICE_ACCOUNT_PATH . Is this the file you refer to?

Reply:
already added in `.env` and  `compose` as discussed, so resolving